package.json

```diff
@@ -5,7 +5,7 @@
     "skuba": "lib/skuba.js"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=14"
```

Comment:
Yeah _this_ is a major. Do you feel it's worthwhile to force people to upgrade at this time?

Reply:
Yeah, let me have another run at this.